guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>Relative links to other guides don't require the <code>/guide</code> text, just <code>..</code>:</p><p> </p><ul><li>text should be <code>ArcGIS Hub Basic</code> - [ArcGIS Hub Basic](../hub-basic-guide)</li><li>[Enteprise Sites] (../enterprise-sites-guide)</li></ul><p> </p><p>The syntax for a note to render it offset from the rest of the text is like this:</p><p> </p><p>&gt;_Note:_ In order to replicate these workflows for yourself, you may have to verify the features available to you, per your ArcGIS Hub licensing.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>